src/parser/druid/restoration/modules/features/Lifebloom.js

```diff
@@ -16,7 +16,8 @@ import { t } from '@lingui/macro';
 
 class Lifebloom extends Analyzer {
   get uptime() {
-    return this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_HOT_HEAL.id);
+    // DTL Lifeblooms are on two targets, getBuffUptime sums both up
+    return this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_HOT_HEAL.id) + (this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_DTL_HOT_HEAL.id) / 2);
```

Comment:
couldn't this make uptime 200% which wouldn't make any sense?

Reply:
I could've reformatted this. Both DTL hots are LIFEBLOOM_DTL_HOT_HEAL, that's why I divided it by 2.
With DTL the first value should be 0, without DTL the second one.